server/conf/routes

```diff
@@ -199,11 +199,11 @@ GET     /logout                      @org.pac4j.play.LogoutController.logout(req
 
 ## Methods for development
 # Seed the database with test content to develop against, and clear the database
-GET     /dev/seed                        controllers.dev.DatabaseSeedController.index(request: Request)
-POST    /dev/seed/canonicalQuestions     controllers.dev.DatabaseSeedController.seedCanonicalQuestions()
+GET     /dev/seed                        controllers.dev.seeding.DatabaseSeedController.index(request: Request)
+POST    /dev/seed/canonicalQuestions     controllers.dev.seeding.DatabaseSeedController.seedQuestions()
```

Comment:
I know it was already like this, but could you move this down so that the routes are more logically grouped while you're here?

Reply:
Done, also renamed `/dev/seed/canonicalQuestions` to `/dev/seedQuestions` for consistency

Reply:
Just noticed we're not really following resource-oriented routing conventions here, but maybe that's fine for dev routes? https://docs.civiform.us/contributor-guide/developer-guide/development-standards#routing-and-controller-methods

Reply:
Yeah, I see a good amount of inconsistency across the `routes` file... wouldn't make it in scope to fix this in this PR.